src/api256.rs

```diff
@@ -362,3 +365,14 @@ pub fn augmentPublicKey256(
             .map_err(WasmError::new)?,
     )
 }
+
+/**
+ * PBKDF2 algorithm which uses SHA-256 as the hashing algorithm. Takes password to create a derived key from,
+ * 32 bytes of salt, and an iteration count and returns a vector of bytes which can be used as a secure derived key.
+ */
+#[wasm_bindgen]
+pub fn pbkdf2SHA256(salt: &[u8], password: &[u8], iterations: u32) -> Vec<u8> {
```

Comment:
Can we just take this as a usize? It'd save a cast and should be the same in JS.

Reply:
Fixed.